pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -150,11 +158,11 @@ func (pl *DefaultPreemption) preempt(ctx context.Context, state *framework.Cycle
 	return bestCandidate.Name(), nil
 }
 
-// FindCandidates calculates a slice of preemption candidates.
```

Comment:
lets move those functions out of the plugin and place them in a separate file so that it is clear which logic is default plugin specific and which can be used by others.